src/VisualStudio/Core/Def/Implementation/ProjectSystem/MiscellaneousFilesWorkspace.cs

```diff
@@ -353,7 +353,10 @@ private ProjectInfo CreateProjectInfoForDocument(string filePath)
 
             var languageServices = Services.GetLanguageServices(languageInformation.LanguageName);
             var compilationOptionsOpt = languageServices.GetService<ICompilationFactoryService>()?.GetDefaultCompilationOptions();
-            var parseOptionsOpt = languageServices.GetService<ISyntaxTreeFactoryService>()?.GetDefaultParseOptions();
+
+            // We only have one code file without project file here, so we cannot find out language version of the project
+            // Use latest language version which is more permissive. https://devdiv.visualstudio.com/DevDiv/_workitems/edit/575761
+            var parseOptionsOpt = languageServices.GetService<ISyntaxTreeFactoryService>()?.GetDefaultParseOptions(useLatestLanguageVersion: true);
```

Comment:
❗️ I would prefer to add the following method to `ParseOptions`:

```csharp
internal abstract ParseOptions WithLatestLanguageVersion();
```

You could then use the following:

```csharp
var parseOptionsOpt = languageServices.GetService<ISyntaxTreeFactoryService>()?.GetDefaultParseOptions()
    .WithLatestLanguageVersion();
```